coffee/lib/abstract-chosen.coffee

```diff
@@ -281,7 +282,15 @@ class AbstractChosen
       return false if /Mobile/i.test(window.navigator.userAgent)
     return true
 
+  @tagname_is_valid: (@form_field) ->
+    if @form_field.tagName.toUpperCase() != 'SELECT'
+      console?.error 'Chosen error: invalid tagName ' + @form_field.selector + ' is not a select' 
+      return false
+    if @form_field.options.length == 0
+      console?.error 'Chosen error: ' + @form_field.selector + ' has no options\'s childs '
+      return false
+    return true
+
   @default_multiple_text: "Select Some Options"
   @default_single_text: "Select an Option"
   @default_no_result_text: "No results match"
-
```

Comment:
You added it back with the two blank spaces -- see [the diff](https://github.com/harvesthq/chosen/pull/2231/files).  There needs to be a new line, no spaces, which should mean nothing showing in the diff for the last line.